library/std/src/process.rs

```diff
@@ -2107,13 +2107,24 @@ pub trait Termination {
 #[stable(feature = "termination_trait_lib", since = "1.61.0")]
 impl Termination for () {
     #[inline]
+    /// Returns `ExitCode::SUCCESS`.
     fn report(self) -> ExitCode {
         ExitCode::SUCCESS.report()
     }
 }
 
 #[stable(feature = "termination_trait_lib", since = "1.61.0")]
 impl<E: fmt::Debug> Termination for Result<(), E> {
+    /// Returns a meaningful exit code, and outputs an error message if
+    /// appropriate.
+    ///
+    /// If the result is `Ok`, returns `ExitCode::SUCCESS`.
+    ///
+    /// If the result is `Err`, writes the [`fmt::Debug`] representation of
+    /// the error value to [`io::stderr`] and returns `ExitCode::FAILURE`.
```

Comment:
I don't want to guarantee this behavior, because I expect us to change it in the future.

In particular, if we add the "generic member access" mechanism, we could then change this trait implementation to attempt to retrieve an `ExitCode` from `E`, and use that if available.